tests/lib/bin/session-tool

```diff
@@ -112,6 +121,18 @@ cat "$tmp_dir/ready.pipe" >/dev/null
 cat >"$tmp_dir/stdin.pipe" &
 cat <"$tmp_dir/stdout.pipe" >&1 &
 cat <"$tmp_dir/stderr.pipe" >&2 &
+(
```

Comment:
could you add a comment that this snippet is specifically so that we can properly support quoting?

Reply:
It's not "properly" but I get your point. (There are more bugs to fix).


I'll do a round of updates to this as I'm actively using it in another branch and add the tests.

Reply:
In the other branch we now have proper quote support!